docs/How-to/Run-a-full-Terra-node/Join-public-network.md

```diff
@@ -49,42 +49,50 @@ You specify the network you want to join by setting the **genesis file** and **s
 
 ### Download the genesis file
 
-You'll need to select the network you want to join and download its `genesis.json` file into your `~/.terra/config` directory. This file specifies the genesis account balances and parameters to use when replaying transactions and syncing.
+Select the network you want to join and download its `genesis.json` file into your `~/.terra/config` directory. This file specifies the genesis account balances and parameters to use when replaying transactions and syncing.
+
+- Columbus-5 mainnet genesis:
+
+```bash
+wget https://columbus-genesis.s3.ap-northeast-1.amazonaws.com/columbus-5-genesis.json
+mv ./columbus-5-genesis.json ~/.terra/config/genesis.json
+```
+- Bombay-12 testnet genesis:
 
 ```bash
-mkdir -p ~/.terra/config
-curl https://columbus-genesis.s3.ap-northeast-1.amazonaws.com/columbus-5-genesis.json > ~/.terra/config/genesis.json
+wget https://raw.githubusercontent.com/terra-money/testnet/master/bombay-12/genesis.json
+mv genesis.json ~/.terra/config/genesis.json
 ```
 
 Note we use the `latest` directory in the [networks repo](https://github.com/terra-money/testnet) which contains details for the latest testnet. If you are connecting to a different testnet, ensure you get the right files.
 
-To verify the correctness of the configuration run:
+To verify your configuration is correct, run:
```

Comment:
"To start terrad"

This doesn't actually verify anything, it just starts terrad.

Reply:
I think this is out of place.